sw/airborne/subsystems/gps.c

```diff
@@ -355,10 +357,15 @@ struct UtmCoor_f utm_float_from_gps(struct GpsState *gps_s, uint8_t zone)
     utm.north = gps_s->utm_pos.north / 100.0f;
     utm.alt = gps_s->utm_pos.alt / 1000.f;
```

Comment:
hmmm... Well I am also inclined here to change the gps to hmsl.

Additionally, I just saw that the conversions from lla to utm don't change the alt from ellipsoid to hsml although utm is defined hsml there.

The current implementation is not coherent. You have strong a opinion on this?